GameData/RealismOverhaul/Engine_Configs/SuperDraco_Config.cfg

```diff
@@ -1,69 +1,121 @@
-//	==================================================
-//	SuperDraco engine.
-
-//	Dimensions: 0.200 x 2.000 m
-//	Gross Mass: 130.40 Kg
-//	Chamber Pressure: 1,000 psi
-//	Throttle Range: 20% to 100%
-//	Burn Time: 300 s
-//	O/F Ratio: 1.30
-
-// FIXME: lots of info missing/unsourced
-
-// SSTU
-// References:
-// [1] http://www.faa.gov/about/office_org/headquarters_offices/ast/media/20140513_DragonFly_DraftEA_Appendices%28reduced%29.pdf
-//	==================================================
+//  ==================================================
+//  SuperDraco global engine configuration.
+
+//  Inert Mass: 65 Kg
+//  Chamber Pressure: 1000 psi
+//  Throttle Range: 20% to 100%
+//  Burn Time: 350 s (minimum)
+//  O/F Ratio: 1.3
+
+//  Sources:
+
+//  FAA - DragonFly RLV test environmental impact: http://www.faa.gov/about/office_org/headquarters_offices/ast/media/20140513_DragonFly_DraftEA_Appendices%28reduced%29.pdf
+//  SpaceX - SuperDraco 3D printed engine chamber: http://www.spacex.com/news/2014/07/31/spacex-launches-3d-printed-part-space-creates-printed-engine-chamber-crewed
+
+//  Used by:
+
+//  * CMES
+//  * SSTU
+
+//  Notes:
+
+//  * Lots of info are missing and/or are unsourced.
+//  ==================================================
+
 @PART[*]:HAS[#engineType[SuperDraco]]:FOR[RealismOverhaulEngines]
 {
-	@title  	  = SuperDraco
-	@manufacturer = SpaceX
-	%description  = A powerful hypergolic engine. Used by the Dragon V2 Command Module for powered landings, trajectory corrections and as a Launch Abort System (LAS). Other applications include hypersonic deceleration and landing for crew and cargo modules.
+	%category = Engine
+	%title = SuperDraco
+	%manufacturer = SpaceX
+	%description = A powerful hypergolic engine. Used by the Dragon V2 Command Module for powered landings, trajectory corrections and as a Launch Abort System (LAS). Other applications include hypersonic deceleration and landing for crew and cargo modules. Diameter: 0.6 m.
+
+	@MODULE[ModuleEngines*]
+	{
+		@minThrust = 14.59
+		@maxThrust = 72.95
```

Comment:
It was set to 85kN because that is the vac thrust, 72kN is sea level thrust as stated in the .pdf